.github/workflows/code_style.yml

```diff
@@ -21,11 +21,12 @@ jobs:
             git
 
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Fetch ${{ github.base_ref }}
         shell: bash
         run: |
+          git config --global --add safe.directory `pwd`
```

Comment:
```suggestion
          git config --global --add safe.directory $(pwd)
```
Better to prefer `$()` over backticks. https://mywiki.wooledge.org/BashFAQ/082

Reply:
I will fix this.

Reply:
Done.

Reply:
Sorry, I meant the other ones as well :sweat_smile:. Also, please use `[ci]` instead of `[github-workflows]`.

Reply:
Done.